make/photon/jobservice/Dockerfile

```diff
@@ -1,13 +1,19 @@
 FROM photon:2.0
 
-RUN mkdir /harbor/ \
-    && tdnf install sudo -y >> /dev/null\
+RUN tdnf install sudo -y >> /dev/null\
     && tdnf clean all \
-    && groupadd -r -g 10000 harbor && useradd --no-log-init -r -g 10000 -u 10000 harbor 
+    && groupadd -r -g 10000 harbor && useradd --no-log-init -r -g 10000 -u 10000 harbor
 
-COPY ./make/photon/jobservice/start.sh ./make/photon/jobservice/harbor_jobservice /harbor/
+COPY ./make/photon/jobservice/harbor_jobservice /harbor/
```

Comment:
We don't need to create `/harbor/` beforehand?

Reply:
`COPY` will create dest dir automatically

Reply:
OK thanks.